### PR TITLE
Update search query simulation script to generate and DB write at least 10M records

### PR DESCRIPTION
**Pull request by christine-w**
Generating 10M records takes anywhere from 20 to 25 minutes on a 15" Macbook Pro with the following specs:
* Processor: 2.8 GHz Intel Core i7
* Memory: 16 GB 2133 MHz LPDDR3

The script used under 200MB of memory during execution (vs over 5GB when trying to run the previous version (PR#5) for 10M records).

### CODE REVIEW

File: simulation/generateSearchQueries.js [hunk 1]
```diff
@@ -5,22 +5,22 @@ const BATCH_SIZE = 1000;
...
 const MAX_LENGTH_OF_STAY = 7;
-const ROOM_TYPES = ['any', 'entire home', 'private', 'shared room'];
+const ROOM_TYPES = ['any', 'Entire home/apt', 'Private room', 'Shared room'];
 
 /* ----- HELPER FUNCTIONS BEGIN ----- */
 
-const incrementIdFrom = (num) => {
-  let startId = num;
+const incrementIdFrom = (startId) => {
+  let id = startId;
```
Comment (tiffaphm) [inline simulation/generateSearchQueries.js]: could this be better written as such? : 
const incrementIdFrom = (id) => {
  return id += 1
}

File: simulation/generateSearchQueries.js [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-const SEARCH_FREQUENCY_IN_SECONDS = 30;
+const SEARCH_FREQUENCY_IN_MS = 100;
 const MARKETS = [
   'San Francisco', 'Seattle', 'Sydney', 'New York', 'Toronto', 'Paris',
   'London', 'Hong Kong', 'Amsterdam', 'Montreal',
 ];
 const MAX_DAYS_UNTIL_TRAVEL = 45;
 const MAX_LENGTH_OF_STAY = 7;
-const ROOM_TYPES = ['any', 'entire home', 'private', 'shared room'];
+const ROOM_TYPES = ['any', 'Entire home/apt', 'Private room', 'Shared room'];
```
Comment (tiffaphm) [inline simulation/generateSearchQueries.js:15]: It's good that you're being more specific to which room types

File: simulation/generateSearchQueries.js [hunk 3]
```diff
@@ -53,29 +54,39 @@ const getRandomRoomType = () =>
...
+const generateQueryBatch = () => {
+  const queries = [];
+  for (let i = 0; i < BATCH_SIZE; i += 1) {
+    const { checkIn, checkOut } = getRandomDateRange();
+    const query = {
+      searchQueryId: nextSearchQueryId(),
+      timestamp: nextSearchTimestamp(),
+      visitId: nextVisitId(),
+      userId: getRandomUser(),
+      market: getRandomMarket(),
```
Comment (tiffaphm) [inline simulation/generateSearchQueries.js:70]: This looks reasonable - will you be showing on your graphs only the data from the SF market, or all markets that were searched?

**Review by tiffaphm — APPROVED** [review]

File: simulation/generateSearchQueries.js (hunk 3 above)
Comment (christine-w) [reply simulation/generateSearchQueries.js:70]: In my Kibana dashboard, I'll probably just start with SF.

File: simulation/generateSearchQueries.js (hunk 2 above)
Comment (christine-w) [reply simulation/generateSearchQueries.js:15]: Yes, it makes sense to match what is coming from the scraped Airbnb data.

**Pull request merged**